.circle/dependencies

```diff
@@ -15,10 +15,10 @@ python --version
 pip --version
 
 # Clone st2 repo so other scripts can reference StackStorm Python code
-git clone --depth 1 --single-branch --branch master https://github.com/StackStorm/st2.git /tmp/st2
+git clone --depth 1 --single-branch --branch ${ST2_BRANCH:-master} https://github.com/StackStorm/st2.git /tmp/st2
```

Comment:
Minor style thing - I would prefer to define the variable with a default value at top of the script (similar as we do for Makefiles) so it's immediately clear which environment variables can be set just by glancing at the first couple of lines of the script.